nautobot/extras/plugins/validators.py

```diff
@@ -40,6 +40,7 @@ def wrap_model_clean_methods():
     """
     Helper function that wraps plugin model validator registered clean methods for all applicable models
     """
-    for model in ContentType.objects.filter(FeatureQuery("custom_validators").get_query()):
-        model_class = model.model_class()
-        model_class.clean = custom_validator_clean(model_class.clean)
+    for app_label, models in FeatureQuery("custom_validators").get_dict():
+        for model in models:
+            model_class = apps.get_model(app_label=app_label, model_name=model)
+            model_class.clean = custom_validator_clean(model_class.clean)
```

Comment:
I _assume_ in this case we don't need the ContentType for some FK lookup in the DB but we just need to wrap the class' clean method, probably better to go straight to the Django for that. Cut out the middle man I _think_.